### PR TITLE
Fixes for #709 and #710

### PR DESCRIPTION
**Pull request by ananyo2012**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] All tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request are descriptively named
* [x] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer
* [x] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by ananyo2012** [comment]
@jywarren Added the fixes. Please review. Also I couldn't wait to see the build-passing icon in the readme :blush:. So I added it in here.

**Comment by ananyo2012** [comment]
Hmm phantomjs is causing real issues in travis-ci. We need to think of a method to cache the phantomjs executable. This is important w.r.t javascript tests. This can be used as a solution to download the executable https://github.com/travis-ci/travis-ci/issues/3225#issuecomment-177592725. Well this another issue. This PR passes by running tests locally.

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -95,10 +95,10 @@ def dashboard
     @comments = @comments.group('DATE(FROM_UNIXTIME(timestamp))') if Rails.env == "production"
     @answer_comments = DrupalComment.joins(:answer, :drupal_users)
                              .order('timestamp DESC')
-                             .where('timestamp - answers.created_at > ?', 0)
+                             .where('timestamp - answers.created_at > ?', 86400)
```
Comment (jywarren) [inline app/controllers/home_controller.rb]: Just curious, what's this value? Could you add a comment at the end of this line, explaining? Tnx!

**Comment by jywarren** [comment]
Hi, @icarito -- could you see Ananyo's comment above? https://github.com/publiclab/plots2/pull/713#issuecomment-241266730 -- ananyo, did you see tests fail but work again when you re-ran them? Or what was the problem? 

Thanks for the `passing` badge, that'll be great to have!

File: app/controllers/home_controller.rb (hunk 1 above)
Comment (ananyo2012) [inline app/controllers/home_controller.rb]: It's the same value used in comments. It is the time difference between the answer creation and comment creation before which it doesn't gets published. Similar to this: https://github.com/publiclab/plots2/blob/master/app/controllers/home_controller.rb#L90. Actually I thought of skipping this off as it is mentioned just before this.

**Comment by ananyo2012** [comment]
@jywarren Yes Jeff the travis-ci build failed at the beginning because the phantomjs installation failed. It passed again after I closed and reopened the PR. You should find ways to cache the phantomjs binary in the build directory so that tests don't fail due to these reasons.

**Comment by icarito** [comment]
El 22/08/16 a las 17:42, Ananya Maiti escribió:
>
> @jywarren <https://github.com/jywarren> Yes Jeff the travis-ci build
> failed at the beginning because the phantomjs installation failed. It
> passed again after I closed and reopened the PR. You should find ways
> to cache the phantomjs binary in the build directory so that tests
> don't fail due to these reasons.
>
Yes, agreed.
If you want to manually restart a build you can repeat the `git push`
command with `--force` argument.

This doesn't change the commits themselves but triggers a travis rebuild.

**Comment by jywarren** [comment]
Rebased and testing...

...all good! Merged in d7333a6ec7480b10bb6855c029a23fb4edbb436a. Thanks!

**Pull request closed without merging**